Doc/conf.py

```diff
@@ -290,7 +290,7 @@
 
 # If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
 # using the given strftime format.
-html_last_updated_fmt = '%b %d, %Y'
+html_last_updated_fmt = '%b %d, %Y (%H:%M, GMT%z)'
```

Comment:
![image](https://github.com/python/cpython/assets/1324225/370d7b62-3a73-45ef-8ca0-14c87dfdd653)


I think it was better without the "+0000", just "GMT" or "UTC" is fine:

```suggestion
html_last_updated_fmt = '%b %d, %Y (%H:%M, GMT)'
```

Reply:
Sphinx uses the UTC timezone to format a date: https://github.com/sphinx-doc/sphinx/blob/b9c85984d2f4863e4c09932c746d915e3233507b/sphinx/util/i18n.py#L192-L199

It should always be +0000, so UTC, no?

Reply:
I realised that the time isn't GMT! E.g. Britain is currently in summer time, so I see:

![image](https://github.com/python/cpython/assets/9087854/e0f50a62-1bc6-4dcc-81a7-c58c1efa3486)

A